frontend/src/js/concept-trees/ConceptTreeNode.tsx

```diff
@@ -107,7 +107,9 @@ const ConceptTreeNode: FC<PropsT> = ({
             ids: [conceptId],
             ...description,
             label: data.label,
-            tables: resetTables(concept?.tables, { useDefaults: true }),
+            tables: concept?.tables
+              ? resetTables(concept.tables, { useDefaults: true })
+              : [],
             selects: resetSelects(concept?.selects, { useDefaults: true }),
```

Comment:
for resetTables you are handling null outside, but I guess in resetSelects you are handling it inside? Would make more sense to go for one or the other approaches.

Reply:
good point, fixed that, handling it always outside now